sdk/identity/identity/README.md

```diff
@@ -1,56 +1,126 @@
-## Azure Identity client library for Javascript
+## Azure Identity client library for JS
 
 This library simplifies authentication against Azure Active Directory for Azure SDK libraries.
 It provides a set of `TokenCredential` implementations which can be passed into SDK libraries
-to authenticate API requests.
+to authenticate API requests. It supports token authentication using an Azure Active Directory [service principal](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli) or [managed identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview).
 
-## Examples
+## Getting started
 
-### Examples for SDK Users
+### Prerequisites
 
-The `@azure/keyvault-keys` library supports the use of `TokenCredential` for authenticating
-requests to Azure Key Vault.  Here's how you would use the `ClientSecretCredential` to authenticate
-SDK calls:
+- an Azure subscription
+  - if you don't have one, you can sign up for a [free account](https://azure.microsoft.com/free/)
+- Node.js 8 LTS or higher
 
-```javascript
-import { KeysClient } from "@azure/keyvault-keys";
-import { ClientSecretCredential } from "@azure/identity";
+**NOTE:** These credential implementations are not supported in the browser yet.  We will provide browser-ready implementations in a future preview release.
 
-const clientId = process.env["CLIENT_ID"] || "";
-const clientSecret = process.env["CLIENT_SECRET"] || "";
-const tenantId = process.env["TENANT_ID"] || "";
-const vaultName = process.env["KEYVAULT_NAME"] || "<keyvault-name>"
+### Install the package
 
-const url = `https://${vaultName}.vault.azure.net`;
-const credential = new ClientSecretCredential(tenantId, clientId, clientSecret)
-const client = new KeysClient(url, credential);
-const getResult = await client.getKey("MyKeyName");
+Install Azure Identity with `npm`:
+```sh
+npm install --save @azure/identity
 ```
 
-If you use the standard environment variable names `AZURE_TENANT_ID`, `AZURE_CLIENT_ID`, and
-`AZURE_CLIENT_SECRET` for storing your AAD client credentials, the previous example can be simplified by using `EnvironmentCredential`:
+## Key concepts
+
+### Credentials
+
+Azure Identity offers a variety of credential classes that are accepted by Azure SDK data plane clients. Each client library documents its Azure Identity integration. Azure SDK management
+libraries do not accept these credentials.
```

Comment:
A way to identify/differentiate the management libraries would be helpful. Maybe mention what they do and they will have either the `@azure/arm-` prefix or the `azure-arm-` prefix ?

Reply:
Good point, I'll add it!